giraffe/src/utils/getMargins.ts

```diff
@@ -11,18 +11,26 @@ export const getMargins = (
   showAxes: boolean,
   xAxisLabel: string,
   yAxisLabel: string,
-  yTicks: number[],
-  yTickFormatter: (tick: number) => string,
+  yTicks: string[] | number[],
+  yTickFormatter: (tick: number | string) => string,
   tickFont: string
 ) => {
   if (!showAxes) {
     return {top: 1, right: 1, bottom: 1, left: 1}
   }
 
-  const longestYTick = maxBy(
-    d => d.length,
-    yTicks.map(t => yTickFormatter(t))
-  )
+  let longestYTick
+  if (typeof yTicks[0] === 'string') {
+    longestYTick = maxBy(
+      d => d.length,
+      (yTicks as string[]).map(t => yTickFormatter(t))
+    )
+  } else {
```

Comment:
do you still need this else? they look pretty similar.. except for typescript stuff.